test/unit/components/join-flow-step.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {mountWithIntl} from '../../helpers/intl-helpers.jsx';
+import JoinFlowStep from '../../../src/components/join-flow/join-flow-step';
+
+describe('JoinFlowStep', () => {
+
+    test('components exist when props present', () => {
+        const props = {
+            children: null,
+            className: 'join-flow-step-class',
+            description: 'description text',
+            headerImgSrc: '/image.png',
+            onSubmit: jest.fn(),
+            nextButton: 'some stuff',
+            title: 'join flow step title',
+            waiting: true
+        };
+        const component = mountWithIntl(
+            <JoinFlowStep
+                {...props}
+            />
+        );
+        expect(component.find('div.join-flow-header-image').exists()).toEqual(true);
+        expect(component.find({src: props.headerImgSrc}).exists()).toEqual(true);
+
+        expect(component.find('.join-flow-inner-content').exists()).toEqual(true);
+        expect(component.find('.join-flow-title').exists()).toEqual(true);
+
+        expect(component.find('.join-flow-title').first()
+            .prop('title')).toEqual(props.title);
+
+        expect(component.find('div.join-flow-description').exists()).toEqual(true);
+        expect(component.find('div.join-flow-description').text()).toEqual(props.description);
+
```

Comment:
this is a tiny quibble, but could you reduce the whitespace some? Seems to me like most of these are similar, and could be in one big bunch! (Like they are in the next test)

Reply:
Sure! Done.